fizzbuzz.js

```diff
@@ -1,9 +1,68 @@
 // This is our main function
 function fizzbuzz() {
-    console.log("Hello, World!");
+    // Iterate from 1 to 255
+    for (let i = 1; i < 256; i++) {
+        // Initialise array
+        let answer = [];
+        if (i % 11 == 0) {
+            // Case where number is a multiple of 11
+            answer.push("Bong");
+        } else {
+            // Append strings for numbers that are identified multiples
+            answer = add_phrase(i, answer, 3, "Fizz");
+            answer = add_phrase(i, answer, 5, "Buzz");
+            answer = add_phrase(i, answer, 7, "Bang");
+        }
 
-    // Put your code here...
+        // Check further multiples
+        answer = insert(i, answer, 13, "Fezz", "B");
+        answer = reverse_phrase(i, answer, 17);
 
+        // Print number or phrase
```

Comment:
Clearly written comments - generally at Softwire we tend to go with the convention to make the code readable without comments, unless there's a particularly non-intuitive bit of logic or context involved. Perhaps you could achieve readability by refactoring the logic to print the correct string into a separate function?